doc/supported-hardware.md

```diff
@@ -22,6 +22,7 @@ The following SR-IOV capable hardware is supported with sriov-network-operator:
 | Mellanox MT28908 Family [ConnectX-6 Dx] | 15b3 | 101d |
 | Mellanox MT28908 Family [ConnectX-6 Lx] | 15b3 | 101f |
 | Mellanox MT2910 Family [ConnectX-7 | 15b3 | 1021 |
+| Mellanox CX8 Family [ConnectX-8 | 15b3 | 1023 |
```

Comment:
NIT: Typo , missing end square "]" bracket.

Reply:
Actually for the CX-7 as well.

Reply:
fixed